packages/react/src/components/InlineLoading/InlineLoading.js

```diff
@@ -28,9 +32,12 @@ export default function InlineLoading({
   const loadingClasses = classNames(`${prefix}--inline-loading`, className);
   const getLoading = () => {
     if (status === 'error') {
-      return <Error20 className={`${prefix}--inline-loading--error`} />;
-    }
-    if (status === 'finished') {
+      return <ErrorFilled16 className={`${prefix}--inline-loading--error`} />;
+    } else if (status === 'warning') {
+      return (
+        <WarningFilled16 className={`${prefix}--inline-loading--warning`} />
+      );
+    } else if (status === 'finished') {
```

Comment:
since the `if` block contains a return statement the `else` is unneeded

```suggestion
    }
    if (status === 'finished') {
```